civihr_default_theme/scss/partials/_button.scss

```diff
@@ -0,0 +1,16 @@
+.btn-default {
+  border-color: $gray;
+  color: $gray;
```

Comment:
`border-color` and `color` are covered by the following sass variables: `$btn-default-color`, `$btn-default-bg`